heroic-core/src/main/java/com/spotify/heroic/consumer/schemas/Spotify100Proto.java

```diff
@@ -104,6 +104,8 @@ public Consumer(
         ingestions.add(ingestion.write(new Ingestion.Request(s, MetricCollection.points(points))));
       }
 
+      reporter.reportMetricsIn(metrics.size());
```

Comment:
We should also count metrics coming in through the non-protobuf serializer, the `Spotify100` schema.

Reply:
Done